<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>759</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QtWebPoster</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>6</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tabHTML">
         <attribute name="title">
          <string>HTML</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="HtmlDocument" name="textEditHTML"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabCSS">
         <attribute name="title">
          <string>CSS</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="CssDocument" name="textEditCSS"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabWeb">
         <attribute name="title">
          <string>WebView</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QWebView" name="webView">
            <property name="url">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabImage">
         <attribute name="title">
          <string>Image View</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QGraphicsView" name="graphicsView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="separator"/>
    <addaction name="actionOpenHTML"/>
    <addaction name="actionSaveHTML"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_CSS"/>
    <addaction name="actionSave_CSS"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionImage_Rendering"/>
    <addaction name="actionEditor_Settings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionStandard_HTML"/>
   </widget>
   <widget class="QMenu" name="menuRender">
    <property name="title">
     <string>Render</string>
    </property>
    <addaction name="actionRenderPage"/>
    <addaction name="actionRenderImage"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuRender"/>
   <addaction name="menuInsert"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenHTML"/>
   <addaction name="actionSaveHTML"/>
   <addaction name="actionRenderPage"/>
   <addaction name="actionRenderImage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRenderPage">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>:/icons/monitor_go.png</normaloff>:/icons/monitor_go.png</iconset>
   </property>
   <property name="text">
    <string>Render Page</string>
   </property>
  </action>
  <action name="actionOpenHTML">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open HTML</string>
   </property>
   <property name="toolTip">
    <string>Open HTML</string>
   </property>
  </action>
  <action name="actionSaveHTML">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save HTML</string>
   </property>
   <property name="toolTip">
    <string>Save HTML</string>
   </property>
  </action>
  <action name="actionRenderImage">
   <property name="icon">
    <iconset theme="image-x-generic">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Render Image</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStandard_HTML">
   <property name="icon">
    <iconset theme="insert-object">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Standard HTML</string>
   </property>
  </action>
  <action name="actionImage_Rendering">
   <property name="text">
    <string>Image Rendering</string>
   </property>
  </action>
  <action name="actionRender_Page">
   <property name="text">
    <string>Render Page</string>
   </property>
  </action>
  <action name="actionOpen_CSS">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open CSS</string>
   </property>
  </action>
  <action name="actionSave_CSS">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save CSS</string>
   </property>
  </action>
  <action name="actionEditor_Settings">
   <property name="icon">
    <iconset theme="preferences-desktop-font"/>
   </property>
   <property name="text">
    <string>Editor Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>HtmlDocument</class>
   <extends>QTextEdit</extends>
   <header>htmldocument.h</header>
  </customwidget>
  <customwidget>
   <class>CssDocument</class>
   <extends>QTextEdit</extends>
   <header>cssdocument.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
